firefox-ios/Shared/Strings.swift

```diff
@@ -1043,6 +1043,11 @@ extension String {
                     value: "Learn more",
                     comment: "This is the learn more text of the Pocket footer on Firefox Homepage.")
             }
+            public static let PositionAccessibilityHint = MZLocalizedString(
+                key: "FirefoxHomepage.Stories.PositionAccessibilityHint.v143",
+                tableName: "FirefoxHomepage",
+                value: "%@ of %@",
+                comment: "On the homepage, in the Stories section, this is the accessibility hint for the position of the current story in the stories stack. The first placeholder is the current position; the second placeholder is the total story count. Example: '1 of 5'")
```

Comment:
```suggestion
                comment: "On the homepage, in the Stories section, this is the accessibility hint for the position of the current story in the stories stack. The first placeholder, %1$@, is the current position; the second placeholder, %2$@, is the total story count. Example: '1 of 5'")
```

ugh, my bad. I forgot about the linter lol let's update this, and then it's good to go in!